pkg/policies/scorecard/scorecard.go

```diff
@@ -0,0 +1,308 @@
+// Copyright 2022 Allstar Authors
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+
+//     http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package scorecard implements the generic Security Scorecards policy
+package scorecard
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/ossf/allstar/pkg/config"
+	"github.com/ossf/allstar/pkg/config/operator"
+	"github.com/ossf/allstar/pkg/policydef"
+	"github.com/ossf/allstar/pkg/scorecard"
+	"github.com/ossf/scorecard/v4/checker"
+	"github.com/ossf/scorecard/v4/checks"
+
+	"github.com/google/go-github/v43/github"
+	"github.com/rs/zerolog/log"
+)
+
+const configFile = "scorecard.yaml"
+const polName = "Security Scorecards"
+
+// OrgConfig is the org-level config definition for this policy.
+type OrgConfig struct {
+	// OptConfig is the standard org-level opt in/out config, RepoOverride
+	// applies to all config.
+	OptConfig config.OrgOptConfig `json:"optConfig"`
+
+	// Action defines which action to take, default log, other: issue...
+	Action string `json:"action"`
+
+	// Checks is a list of check names to run from Security Scorecards. These
+	// must match the name that the check uses in it's call to
+	// "registerCheck". See the check code for each name:
+	// https://github.com/ossf/scorecard/tree/main/checks For example, the name
+	// for the Signed Releases check is "Signed-Releases".
+	Checks []string `json:"checks"`
+
+	// Threshold is the score threshold that checks must meet to pass the
+	// policy. If all checks score equal or above the threshold, the Allstar
+	// policy will pass. The default is checker.MaxResultScore:
+	// https://pkg.go.dev/github.com/ossf/scorecard/v4@v4.4.0/checker#pkg-constants
+	Threshold int `json:"threshold"`
```

Comment:
Allow a threshold to be set for each check? Is there a use case for this?

Reply:
Don't want to overcomplicate now. If we have a usecase then we can add it.